### PR TITLE
Fix Posts block layout

### LINKED ISSUE #1728: ISBAT preview Post Grid with images within the Editor
### Describe the bug:
ISBAT see the feature images assigned to posts when the Posts Grid block is inserted into the editor

### To reproduce:

1. Ensure your site has at least 1 (but make it a couple) of posts with a featured image assigned.
2. In a new page insert a 'Posts' block which includes the posts with the featured images.

### Expected behavior:
I would expect to see the preview of the block in the editor include feature images of posts

### Screenshots:

Screen Shot 2020-10-11 at 9 46 00 pm

The top two posts in this list have featured images assigned - but they are not displayed.

### Isolating the problem:

- [x] This bug happens with no other plugins activated
- [x ] This bug happens with a default WordPress theme active
- [x ] This bug happens **without** the Gutenberg plugin active
- [x ] I can reproduce this bug consistently using the steps above

### WordPress version:
5.5.1

### PR DESCRIPTION
**Pull request by AnthonyLedesma**
### Description

Closes #1728 
API changes introduced in https://github.com/godaddy-wordpress/coblocks/pull/1576 caused an unintentional change in property syntax. Minor change within the `edit.js` file to allow images to properly render and controls along with them.

In addition to the syntax error when accessing properties, the `coblocks/posts` block also has a failure to properly apply gutter attributes and styles from the `Gutter Controls` component. 

Changes here properly apply attributes, styles, and classes to the `coblocks/posts` SSR block.

postsBlockImagesLayout

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by AnthonyLedesma** [comment]
Made some progress with proper supports registration. I had failed to pass the supports from the block.json into the block registration here.
https://github.com/godaddy-wordpress/coblocks/blob/c43f9ac285dc207d61b3c8ee8df5931368dded5c/src/blocks/posts/index.php#L324-L325

File: src/components/gutter-control/gutter-wrapper.php [hunk 1]
```diff
@@ -33,10 +33,60 @@ function coblocks_add_gutter_styles( $styles, $attributes ) {
...
+function coblocks_register_gutter_support( $block_type ) {
+	$has_gutter_support = false;
+
+	if ( property_exists( $block_type, 'supports' ) ) {
+
+		$has_gutter_support = coblocks_experimental_get( $block_type->supports, array( 'gutter' ), false );
+
+	}
+
+	if ( $has_gutter_support ) {
```
Comment (jonathanbardo) [inline src/components/gutter-control/gutter-wrapper.php:58]: @AnthonyLedesma I would avoid the big if condition here and reverse and return early.
```php
if (  ! $has_gutter_support ) {
  return
}
```

File: src/utils/utils.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * Utility is cloned from source: 
+ * https://github.com/WordPress/gutenberg/blob/5f614054e92f5147018f07db601ac1c5eae9d14e/lib/utils.php#L8-L33
+ *
+ * @param array $array    An array from where we want to retrieve some information from.
+ * @param array $path     An array containing the path we want to retrieve.
+ * @param array $default  The return value if $array or $path is not expected input type.
+ *
+ * @return array An array matching the path specified.
+ */
+function coblocks_experimental_get( $array, $path, $default = array() ) {
```
Comment (jonathanbardo) [inline src/utils/utils.php:22]: @AnthonyLedesma Any reason why we couldn't just use the Gutenberg function directly?

**Review by EvanHerman — APPROVED** [review]
Works well on my end 👍 

I pushed one small change based on @jonathanbardo comment regarding the conditional.

**Pull request merged**